presentations/2023-02-01_what-is-data-science/index.qmd

```diff
@@ -1,10 +1,24 @@
 ---
-title: "Everything you ever wanted to know about data science (but were too afraid to ask)"
-format: revealjs
-from: markdown+emoji
-author: Chris Beeley
+title: Everything you ever wanted to know about data science
+subtitle: but were too afraid to ask
+author: "[Chris Beeley](mailto:chris.beeley1@nhs.net)"
+date: 2023-08-02
+date-format: "MMM D, YYYY"
```

Comment:
Can the date format be `D MMM, YYYY`?

Reply:
Yes it can, I have no opinion

Reply:
I just stole the date format from @ChrisBeeley 😝 I think the key thing is consistency - what do we prefer? I feel like I also don't have an opinion, so happy to be guided by whatever someone thinks is the best. For reference, [here is how it looks](https://the-strategy-unit.github.io/data_science/presentations/2023-07-11_haca-nhp-demand-model/#/title-slide)